plugins/dapustor/dapustor-nvme.c

```diff
@@ -312,16 +312,16 @@ static void show_dapustor_ext_add_smart_log_jsn(struct nvme_extended_additional_
 	json_object_add_value_object(dev_stats, "temp_since_born", entry_stats);
```

Comment:
I assume you want to keep the `temp_since_born` here and not `temp_since_bootup`?

Reply:
both 'temp_since_born' and 'temp_since_bootup' need to be displayed.
the previous problem was that 'temp_since_born' was duplicated and 'temp_since_bootup' was missing.